packages/fxa-payments-server/server/lib/server.test.js

```diff
@@ -1,3 +1,53 @@
-it('works', () => {
-  expect(true).toEqual(true);
+const request = require('supertest');
+const server = require('./server')();
+const app = server.app;
+const pkg = require('../../package.json');
+
+function expectValueNotToBeUnknown(value) {
+  expect(value).toBeTruthy();
+  expect(value).not.toBe('unknown');
+}
+
+describe('Test simple server routes', () => {
+  test('__version__ should have correct structure', done => {
```

Comment:
Only nit would be that we use `it()` elsewhere rather than `test()` - but it's not a showstopper.